CHANGELOG.md

```diff
@@ -24,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `extend(DerivationPath)` extends and returns DescriptorPublicKey
     - `as_string()` returns DescriptorPublicKey as String
   - Add to `interface Blockchain` the `get_height()` and `get_block_hash()` methods [#184]
+  - Add to `interface TxBuilder`  the `set_recepients(recipients: Vec<AddressAmount>)` method [#186] 
```

Comment:
Typo here on the word "recipient" in set_recipients:
`set_recipients(recipients: Vec<AddressAmount>)` method

Reply:
thank you thunder.